.gitignore

```diff
@@ -0,0 +1 @@
+node_modules/
```

Comment:
- [ ] Kindly add `test.md` to `.gitignore` as mentioned in the requirements so the file doesn't get pushed to the remote repo 👍

Reply:
@ham-manel thanks